LICENSE.txt

```diff
@@ -39,11 +39,11 @@ I grant permission for some parts of FML to be redistributed outside the terms o
 the minecraft modding community. All contributions to these parts should be licensed under the same additional grant.
 
 -- Runtime patcher --
-License is granted to redistribute the runtime patcher code (src/main/java/net/minecraftforge/fml/common/patcher
+License is granted to redistribute the runtime patcher code (src/main/java/net/minecraftneoforge/fml/common/patcher
 and subdirectories) under any alternative open source license as classified by the OSI (http://opensource.org/licenses)
 
 -- ASM transformers --
-License is granted to redistribute the ASM transformer code (src/main/java/net/minecraftforge/common/asm/ and subdirectories)
+License is granted to redistribute the ASM transformer code (src/main/java/net/minecraftneoforge/common/asm/ and subdirectories)
```

Comment:
This entire section is stale - no such folder exists https://github.com/neoforged/NeoForge/tree/1.20.x/src/main/java/net/minecraftforge/common/asm is a 404

It may be intended to refer to the coremods we provide at https://github.com/neoforged/NeoForge/tree/1.20.x/src/main/resources/coremods

Which would mean it needs to mention `/src/main/resources/coremods`

Reply:
I replaced this section with Coremods and updated the link with newest commit